index.html

```diff
@@ -28,6 +28,218 @@ <h1 class="headline-title">Hey There.<br>I’m Alex</h1>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="work-tag-group">
+            <li class="work-tag-dark">html</li>
+            <li class="work-tag-dark">bootstrap</li>
+            <li class="work-tag-dark">Ruby</li>
+          </ul>
+        </div>
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark">
+            See Project
+          </a>
+        </div>
+      </article>
+      <article class="work-container bg-image">
+        <div class="work-content-dark">
+          <p class="work-title dark">
+            Profesional Art Printing Data
+          </p>
+          <p class="work-text dark">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="work-tag-group">
+            <li class="work-tag-dark">html</li>
+            <li class="work-tag-dark">bootstrap</li>
+            <li class="work-tag-dark">Ruby</li>
+          </ul>
+        </div>
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark">
+            See Project
+          </a>
+        </div>
+      </article>
+      <article class="work-container bg-image">
+        <div class="work-content-dark">
+          <p class="work-title dark">
+            Profesional Art Printing Data
+          </p>
+          <p class="work-text dark">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="work-tag-group">
+            <li class="work-tag-dark">html</li>
+            <li class="work-tag-dark">bootstrap</li>
+            <li class="work-tag-dark">Ruby</li>
+          </ul>
+        </div>
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark">
+            See Project
+          </a>
+        </div>
+      </article>
+      <article class="work-container bg-image">
+        <div class="work-content-dark">
+          <p class="work-title dark">
+            Profesional Art Printing Data
+          </p>
+          <p class="work-text dark">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+          </p>
+          <ul class="work-tag-group">
+            <li class="work-tag-dark">html</li>
+            <li class="work-tag-dark">bootstrap</li>
+            <li class="work-tag-dark">Ruby</li>
+          </ul>
+        </div>
+        <div class="work-footer-container-dark">
+          <a class="work-button button dark">
+            See Project
+          </a>
+        </div>
+      </article>
+    </section>
+    <section class="about-container">
+      <div class="about-content">
+        <p class="about-title">About me</p>
```

Comment:
- [ ] Kindly use a heading element (ie `hx` element => `<h2>` or `<h3>` ... or `<h5>`) to wrap the title of this section. It is best practice for HTML semantics.